iModelJsNodeAddon/api_package/ts/src/test/DgnDb.test.ts

```diff
@@ -510,4 +510,23 @@ describe("basic tests", () => {
       }],
     });
   });
+
+  it("testConvertEC2Schemas", async () => {
+    const schemaXml = `<?xml version="1.0" encoding="UTF-8"?>
+      <ECSchema schemaName="TestSchema" version="1.0" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.2.0">
+        <ECSchemaReference name="RefSchema" version="01.00" prefix="rs" />
+        <ECClass typeName="TestEntityClass" isDomainClass="true" />
+      </ECSchema>`;
+
+    const refSchema = `<?xml version="1.0" encoding="UTF-8"?>
+      <ECSchema schemaName="RefSchema" nameSpacePrefix="rs" version="1.0" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.2.0">
+        <ECClass typeName="TestStructClass" isStruct="true" />
+      </ECSchema>`;
+
+    assert.isTrue(dgndb.isOpen());
+    const ec3Schemas: string[] = dgndb.convertEC2XmlSchemas([schemaXml, refSchema], { schemaLockHeld: false });
+    assert.equal(ec3Schemas.length, 2);
+    dgndb.saveChanges();
+    dgndb.closeIModel();

```

Comment:
I did not include ConversionSchema in the schema context for the conversion because I did not see it used in the DgnV8 connector source code during schema conversion.